### PR TITLE
Added draft updated format governance

### PR DESCRIPTION
**Pull request by justinbuck**

### CODE REVIEW

**Comment by justinbuck** [comment]
I need to fix typos etc

**Review by vturpin — APPROVED** [review]

**Comment by vturpin** [comment]
I've corrected some typos.

File: OG format governace [hunk 1]
```diff
@@ -17,20 +17,20 @@ The format will need to grow and evolve with time while being managed effectivel
 
 1. The team will analyze the issue formally. If the issue is not detailed enough or if there is no example associated with the issue, the issue will be rejected. If the issue is accepted for review, then the team will study it and provide an answer. The issue can lead to a modification of the format or not. If there is no consensus on the solution, the suggested solution will be rejected and the format will remain unchanged. However, a new branch can be created to demonstrate the benefit of the solution. The same issue can be submitted again with a demo.
 
-2. If a format update is required, then a pull will be made from the repository and updates made to the format description and supporting tools such as format checkers. The draft changes will go to OGDMTT members within the GitHub repository for appraval. Stakeholers will be noftifed of the change at this point.
+2. If a format update is required, then a pull will be made from the repository and updates made to the format description and supporting tools such as format checkers. The draft changes will go to OGDMTT members within the GitHub repository for approval. Stakeholers will be noftifed of the change at this point.
```
Comment (vturpin) [inline OG format governace:21]: How do we guaranty that "Stakeholers will be noftifed of the change" ?

File: OG format governace [hunk 2]
```diff
@@ -17,20 +17,20 @@ The format will need to grow and evolve with time while being managed effectivel
...
+The vocabualry collections are more dynamic than the format. When terms are added to vocabualry collections there will be no need to update the format with the vocabulary contected being governed by a dedicate GitHub page for the collection.
 
-When new vocabularies are required or existing fields in teh format need to be constrained against a vocabualry then a simialr process as descibed for issues will be followed. There are additional options available for vocabualries:
-1. Identify an existing voabculary that already meets the needs of the OceanGliders community
+When new vocabularies are required or existing fields in the format need to be constrained against a vocabulary then a simialr process as descibed for issues will be followed. There are additional options available for vocabualries:
+1. Identify an existing vocabulary that already meets the needs of the OceanGliders community
 2. Create a scheme on an existing vocabulary where terms are too diverse to meet the needs of the OceanGliders community
```
Comment (vturpin) [inline OG format governace:27]: Shall we describe what a scheme is ?

**Review by vturpin — COMMENTED** [review]
Some questions about format governance

File: OG format governace (hunk 1 above)
Comment (justinbuck) [reply OG format governace:21]: I think we need to leverage the github/open science framework, may need some expert help from @soerenthomsen  on how to achieve this

File: OG format governace (hunk 2 above)
Comment (justinbuck) [reply OG format governace:27]: Link to the definition here?
https://www.bodc.ac.uk/resources/products/web_services/vocab/#uris

File: OG format governace (hunk 1 above)
Comment (callumrollo) [reply OG format governace:21]: If stakeholders are individuals/groups who have contributed to the repo, they will be notified of new releases, pull requests etc.It's up to the individual to decide what github notifications they recieve. Alternatively, if you know who the stakeholders are that you wish to notify, you can @ them like @vturpin to attract their attention to a change.

File: OG format governace (hunk 2 above)
Comment (callumrollo) [reply OG format governace:27]: That link is useful. Once it's added I think this PR is good to merge

**Review by callumrollo — CHANGES_REQUESTED** [review]

**Comment by castelao** [comment]
Thanks @justinbuck for putting this together. I have a couple of suggestions. I'll work on this tomorrow.

**Comment by castelao** [comment]
I'm working on some suggestions, PR #87 . If accepted, it would modify this PR here. It is still a draft.

I have a question. Have we ever defined which spelling are we going to use? It is probably worth defining one for consistency.

**Comment by castelao** [comment]
I propose some changes to the 'Who is reviewing the issues?'. I think it is important to define it well. Currently, we have the list of people that were participating in the regular meetings since I don't remember anymore (An acknowledgment here that there were plenty of valuable contributions from people not participating in those meetings, and we shall not forget that or omit that in the document.) We needed a way to not only review for errors or missed potential issues but also to decide what would be a good direction for the community. The recent #103 is a good example of that sort of strategic decision. While it made sense when we moved to GitHub to keep the same people that were deciding before in our synchronous remote meetings, it is probably time to evolve.

I do think it is required a formal committee to make the decisions and take the responsibility for those, but I also think it can be beneficial to rotate it. If there is interest, I can write the first draft on that topic to be added here, but I had in mind something  like:
- 2 year term with the possibility of one time renewal for each member.
- A diverse committee (geographical region, type of gliders, genre, ...).
- Some sort of voting system. I'm not sure yet of the best way for that.
- Changes in the format would require the approval of at least 3 members of this committee. Just one veto would be enough to put a change on hold for further discussions.
- Committee shall be formed by experts on underwater gliders (it requires technical and practical knowledge), and use their expertise to represent the community. The committee shall engage the community to express their opinion on discussing issues/PRs, and while the final decision is from the committee voting, this committee shall keep in mind that it is not a personal vote, but instead, they shall use their expertise to represent the community.
- The rotation shall not be done at once but in alternating chunks, so we always have some seasoned members to transfer experience.

File: governance.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# OceanGliders format goverance proposal
+
+## Status
+
+2022-06-27 draft policy based on the existing readme for review and feeedabck
+from the OGDMTT and OG steering team
```
Comment (justinbuck) [inline governance.md:6]: Need to define the OGDMT acronym

File: governance.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+1. The team will analyze the issue formally. If the issue is not detailed enough or if there is no example associated with the issue, the issue will be rejected. If the issue is accepted for review, then the team will study it and provide an answer. The issue can lead to a modification of the format or not. If there is no consensus on the solution, the suggested solution will be rejected and the format will remain unchanged. However, a new branch can be created to demonstrate the benefit of the solution. The same issue can be submitted again with a demo.
+
+2. If a format update is required, then a pull will be made from the repository and updates made to the format description and supporting tools such as format checkers. The draft changes will go to OGDMTT members within the GitHub repository for approval. Stakeholers will be noftifed of the change at this point.
+
+## Vocabularies
+The vocabulary collections are more dynamic than the format. When terms are added to vocabulary collections there will be no need to update the format with the vocabulary contected being governed by a dedicate GitHub page for the collection.
```
Comment (justinbuck) [inline governance.md:25]: Need to develop the introduction to make it accessible for non-experts

Comment (justinbuck) [reply governance.md:25]: Vocabularies tend to be managed via the hosting vocabulary services and governance teams

Comment (justinbuck) [reply governance.md:25]: Section on adding new terms to existing vocabularies

Comment (justinbuck) [reply governance.md:25]: Add in that people need to work collaboratively with the OGDMTT

**Comment by justinbuck** [comment]
@gwemon , Dani Wright,  could you review this please, @emmerbodc and i have revise the text

**Comment by gwemon** [comment]
> @gwemon , Dani Wright, could you review this please, @emmerbodc and i have revise the text

Thanks Justin. A lot of typos to be corrected but I guess you know that. Otherwise here are a few comments and/or suggestions: 
1- Use OGDMTT consistently once it's been defined 
2- In the "Data Format" section you're not saying how somebody goes about reporting an issue. Is it done by raising a Github issue ticket?
3- The OGDMTT members will evaluate the issue formally... when? at monthly meetings? on reception?
4- Clarify the sentence starting with: "If there is no consensus on the solution" to "If there is no consensus on the solution to the issue raised, no change will be made to the format." then continue as is.
5- "pull request" instead of just "pull"?
6- "A list of the vocabulary collections... IS specified...
7- review sentence starting with "OceanGliders use collections hosted by various organisations..."  for clarity. Not sure I got this right but here is a suggestion: "OceanGliders use controlled vocabularies that are hosted by various organisations and countries. These vocabularies are managed on behalf of communities working together to ensure that many users can re-use the content, suggest improvements, or propose new terms. It is recommended that glider users work with the governance groups listed in the format documentation (link) and follow the governance process whenever new terms or corrections are needed."

**Review by vturpin — APPROVED** [review]

**Review by callumrollo — APPROVED** [review]

**Pull request merged**